stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -59,29 +59,57 @@ public struct DoubleWidth<Base : FixedWidthInteger>
   internal var _storage: (low: Low, high: High)
 #endif
 
+  /// The high part of the value.
   @_inlineable // FIXME(sil-serialize-all)
+  @_transparent
   public var high: High {
     return _storage.high
   }
 
+  /// The low part of the value.
   @_inlineable // FIXME(sil-serialize-all)
+  @_transparent
   public var low: Low {
     return _storage.low
   }
 
+  /// Creates a new instance from the given tuple of high and low parts.
+  ///
+  /// - Parameter value: The tuple to use as the source of the new instance's
+  ///   high and low parts.
   @_inlineable // FIXME(sil-serialize-all)
-  public // @testable
```

Comment:
Although it appears that this initializer was originally not intended for public consumption, [the documentation](https://github.com/apple/swift/pull/13784/files#diff-1a1ccf1c510bfc49a19be3eb3c378a8fL29) tells users to use it. In fact, this initializer is promoted as part of the main use case for `DoubleWidth` (and I would agree with that characterization).